### ISSUE TITLE
Extra new lines appended to variables stanzas

### ISSUE THREAD

**Issue by u2mejc** [hashicorp/hcl#161]
Please let me know if this is the appropriate location for this issue. 👍 
### HCL Template

``` hcl
variable "vpc_id" {}
variable "env" {}
variable "db_parameter_group" {}
variable "publicly_accessible" {}
```
### Expected behavior

One line variables should not require a tailing new line when the next stanza is another single line variable, just like using assigning variables inside a resource doesn't add tailing new lines.
### Actual behavior

Using tools such as `terraform fmt` cause a tailing newline to be appended to variable lists, doubling their length, padding the file, and forcing excessive scrolling.
### Steps to reproduce
- Create file:

``` hcl
variable "vpc_id" {}
variable "env" {}
variable "db_parameter_group" {}
variable "publicly_accessible" {}
```
- Run `terraform fmt`
- Output diff:

``` diff
 variable "vpc_id" {}
+
 variable "env" {}
+
 variable "db_parameter_group" {}
+
 variable "publicly_accessible" {}
```

**Comment by stibi** [comment]
Hi, I'd also like to ask if this variables formatting is really expected? It looks a bit weird.

**Comment by grubernaut** [comment]
Related to: #113 
cc: @mitchellh

**Comment by mitchellh** [comment]
Fixed this particular case! :D See #184.
